README.md

```diff
@@ -28,3 +28,127 @@ After matching the slide filenames against the sample IDs from the clinical data
+* Trained models saved at TensorFlow checkpoints.
+
+**Example**
+```
+python model_train.py -m image_genome --ngf 80 --gn y --ns 6
+```
+This command will train the GSCNN model with 80 genomic features where normalization is applied starting from column 6.
+
+## Testing
+
+The command
+```
+python model_test.py
+```
+will generate risk values for testing patients using a trained model. The testing input parameters and their default values for the model_test.py can be listed using the command
+```
+python model_test.py -h
+```
+that generates output
+```
+usage: model_test.py [-h] [-m M] [--kp KP] [--bs BS] [-i I] [-r R] [-t T]
+                     [-d D] [-f F] [--ic IC] [--ngf NGF] [--gn GN] [--ns NS]
+                     [--nm NM]
+
+Arguments for prediction using an SCNN/GSCNN model
+
+optional arguments:
+  -h, --help  show this help message and exit
+  -m M        image_only or image_genome; image_only for SCNN and image_genome
+              for GSCNN. The default is image_only
+  --kp KP     Keeping probability for Test time dropout. Default value = 1.
+  --bs BS     batch size. Default value = 14.
+  -i I        Path to the folder containing ROI .pngs. The default path is ../images/test
+  -r R        Path where testing results will be generated. The default path is ./test_results
+  -t T        Path to the training meta files. The default is ./checkpoints
+  -d D        Path where the temporary binary files for testing will be generated. The default path is ./tmp
+  -f F        Path to the features data. The default path is ./inputs/all_dataset.csv
+  --ic IC     indexes column in the ./inputs/all_dataset.csv file, starting
+              from 0. Default value = 1
+  --ngf NGF   Number of genomic features. Default value = 2.
+  --gn GN     Z-score normalization of the genomic features; y for yes and n for no.
+              Default value = n.
+  --ns NS     Starting column number of Normalization for the genomic values
+              we want to do the Z normalization. This is the column number in
+              the ./inputs/all_dataset.csv file, starting from 0. Default
+              value = 6; Note: The columns we want to normalize should get assigned
+              as the last columns of the all_data.csv file
+  --nm NM     number epochs for model averaging. Default value = 5.
+``` 
+The outputs generetaed by model_test.py are:
+ 
+* Input binary files saved in ./tmp folder by default
+* ###_testset_cindex.txt saved in the folder specified by the -r argument. ### is the date/time when the executable was run. This file contains the final C-Index of the model during testing. As described in the paper, this C-Index represents the accuracy of model-averaged risks (the last 5 models by default that are saved during training). [See the *Methods* section for more details](http://www.pnas.org/content/early/2018/03/09/1717139115).
```

Comment:
I modified this part significantly - please check for accuracy.

Reply:
Checked- I just made a minor change to make it more clear.